src/stories/material-description/material-description.scss

```diff
@@ -0,0 +1,17 @@
+.material-description {
+  max-width: 750px;
+  padding: 16px;
+  padding-top: 60px;
+  padding-bottom: 44px;
```

Comment:
Just a note: As we talked about earlier I think we should have a general task looking into making sass spacing variables we can use in these cases.